app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -292,10 +294,17 @@ fun AppViewContent(
       )
     }
     if (app.isAppCoins) {
+      val bonusTag = "bonus-banner"
```

Comment:
I see these string literals repeated 7 times and all are connected with `getBonusIconLeft`.
Let's extract those as a reusable component to encapsulate these constants and not repeat them.